app/helpers/application_helper.rb

```diff
@@ -435,4 +435,11 @@ def hidden_spinner(text = '', options = {})
   def hosts_count(resource_name = controller.resource_name)
     @hosts_count ||= Host::Managed.reorder('').authorized.group("#{resource_name}_id").count
   end
+
+  def webpack_dev_server
+    return unless Rails.env.development?
+    host = ::Rails.configuration.webpack.dev_server.host.call
```

Comment:
Hmmm I don't think it's this line causing it but regardless - host.call won't return 0.0.0.0